src/Azure.Functions.Cli/Helpers/WorkerRuntimeLanguageHelper.cs

```diff
@@ -25,7 +25,7 @@ public static class WorkerRuntimeLanguageHelper
         private static readonly IDictionary<WorkerRuntime, IEnumerable<string>> availableWorkersRuntime = new Dictionary<WorkerRuntime, IEnumerable<string>>
         {
             { WorkerRuntime.dotnet, new [] { "c#", "csharp", "f#", "fsharp" } },
-            { WorkerRuntime.dotnetIsolated, new [] { "dotnet-isolated" } },
+            { WorkerRuntime.dotnetIsolated, new [] { "dotnet-isolated", "c#-isolated", "csharp-isolated", "f#-isolated", "fsharp-isolated" } },
```

Comment:
Do we still need `dotnet-isolated` in the array containing `c#-isolated` and `csharp-isolated` here?

Reply:
I'll have to have a second look at it, but I _think_ the reason I kept `dotnet-isolated` in there was for backwards compatibility, as prior to this change that's how things were identified.

Reply:
I've had a bit of a poke around and I don't think it can be removed with a lot of other cascading changes.

Removing `dotnet-isolated` will cause existing projects to fail to launch using `func start` as it's unable to detect the worker type. I also tested changing the worker type in `local.settings.json` to `c#-isolated` but that then errors within the WebJobs dependency as it couldn't find the functions language `c#-isolated`, so there'd be a downstream impact.

Additionally, all templates would need to be updated to either say `c#-isolated` or `f#-isolated` in them before the change would usable.

Ultimately, the additional worker types are probably not doing anything of value at present, they are more future proofing 😅